packages/graphback-cli/src/components/db.ts

```diff
@@ -13,57 +14,62 @@ const handleError = (err: { code: string; message: string; }): void => {
     logError(err.message)
   }
   process.exit(0)
-}
+};
 
 export const createDBResources = async (cliFlags: { project?: string }): Promise<any> => {
   let databaseOperations: any;
   try {
     const config = await loadConfig({
       rootDir: process.cwd(),
-      extensions: [graphbackConfigExtension]
+      extensions: [graphbackConfigExtension, dbMigrationConfigExtension]
     });
-    const project = config.getProject(cliFlags.project || 'default')
-    const graphbackConfig = project.extension(extensionName);
+    const project = config.getProject(cliFlags.project || 'default');
+    const graphbackConfig = project.extension(graphbackExtension);
+    const dbMigrationConfig = project.extension(dbmigrationsExtension);
 
     if (!graphbackConfig) {
-      throw new Error(`You should provide a valid '${extensionName}' config to generate schema from data model`);
+      throw new Error(`You should provide a valid '${graphbackExtension}' config to generate schema from data model`);
+    }
+
+    if (!dbMigrationConfig) {
+      throw new Error(`You should provide a valid '${dbmigrationsExtension}' config to migrate schema`);
     }
 
     if (!graphbackConfig.model) {
-      throw new Error(`'${extensionName}' config missing 'model' value that is required`);
+      throw new Error(`'${graphbackExtension}' config missing 'model' value that is required`);
     }
 
-    const models = new GlobSync(`${graphbackConfig.model}/*.graphql`)
+    const models = new GlobSync(`${graphbackConfig.model}/*.graphql`);
```

Comment:
So I had this idea that if we migrate dbMigrationConfig then we do not need graphbackConfig for migration. 

Why? We can use the top-level schema as input instead. config.getSchema() method will be much better here as db migration package is not specific to graphback and it can be used outside graphback to do database migrations

Reply:
I will remove graphbackConfig from this file. Wasn't sure whether I need to keep it, but now I know